index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env ./node_modules/.bin/babel-node
+
+
+import nomnom from 'nomnom';
+import glob from 'glob';
+import path from 'path';
+
+
+// Find all available scaffolds
```

Comment:
Just a bit of automation here. Not sure how valuable it is. We can always hardcode this like:
```js
import package from './scaffold/package;
const scaffolds = {
  package
};
```
But have to add new scaffolds manually here every time.

Let me know which way you prefer.